go/worker/compute/txnscheduler/service.go

```diff
@@ -15,9 +16,13 @@ func (w *Worker) SubmitTx(ctx context.Context, rq *api.SubmitTxRequest) (*api.Su
 		return nil, api.ErrUnknownRuntime
 	}
 
+	rtid, _ := rq.RuntimeID.MarshalText()
+	_ = pprof.WriteHeap("multiple-runtimes.worker_" + string(rtid) + ".beforeSubmitRuntimeTx")
```

Comment:
So every SubmitTx call is going to write garbage to the filesystem?

Reply:
No, this was just for playing around. I will probably do either:
- enable pprof service so `oasis-test-runner` can poll pprof info from the worker when needed, or
- push pprof heap each time metrics to prometheus or jaeger are pushed.

Reply:
The prometheus docs says it should *not* be used as a logging tool, so passing pprof as prometheus label is wrong: https://prometheus.io/docs/introduction/faq/#how-to-feed-logs-into-prometheus

Searching on...

Reply:
Dumping files to disk has now been removed. I left the `DumpHeapToFile()` in, if we decide to use it in the future.